src/import-data/ImportDataOverview.ts

```diff
@@ -75,7 +75,7 @@ interface ImportDataOverviewProps {
   isProtectedData: boolean;
   snapshotResponses: { status: string; message: string | undefined }[] | undefined;
   snapshots: { id: string; title: string }[];
-  url: string;
+  url?: string;
```

Comment:
This type was incorrect, but the query parameters typed as `any` prevented TS from catching it.